cmd/rbac/rbac_describe_permissions.go

```diff
@@ -0,0 +1,111 @@
+package rbac
+
+import (
+	"github.com/spf13/cobra"
+	"gitlab.com/stackvista/stackstate-cli2/generated/stackstate_api"
+	"gitlab.com/stackvista/stackstate-cli2/internal/common"
+	"gitlab.com/stackvista/stackstate-cli2/internal/di"
+	"gitlab.com/stackvista/stackstate-cli2/internal/printer"
+)
+
+type DescribePermissionsArgs struct {
+	Subject    string
+	Permission string
+	Resource   string
+}
+
+type Permissions map[string][]string
+
+func DescribePermissionsCommand(deps *di.Deps) *cobra.Command {
+	const (
+		PermissionUsage = "Filter the permissions by permission name"
+		ResourceUsage   = "Filter the permissions by a resource identifier (e.g. system or a view name)"
+	)
+
+	args := &DescribePermissionsArgs{}
+	cmd := &cobra.Command{
+		Use:   "describe-permissions",
+		Short: "Show permissions of a subject",
+		Long:  "Show permissions of a subject.",
+		RunE:  deps.CmdRunEWithApi(RunDescribePermissionsCommand(args)),
+	}
+
+	cmd.Flags().StringVar(&args.Subject, Subject, "", SubjectUsage)
+	cmd.MarkFlagRequired(Subject) //nolint:errcheck
+
+	cmd.Flags().StringVar(&args.Permission, Permission, "", PermissionUsage)
+	cmd.Flags().StringVar(&args.Resource, Resource, "", ResourceUsage)
+
+	return cmd
+}
+
+func RunDescribePermissionsCommand(args *DescribePermissionsArgs) di.CmdWithApiFn {
+	return func(
+		cmd *cobra.Command,
+		cli *di.Deps,
+		api *stackstate_api.APIClient,
+		serverInfo *stackstate_api.ServerInfo,
+	) common.CLIError {
+		description, resp, err := api.PermissionsApi.DescribePermissions(cli.Context, args.Subject).Execute()
```

Comment:
You should be able to pass the `permission` and `resource` flags to the Rest request as request parameters to do the filtering server-side.

```scala
        path(Segment) { handle =>
          get {
            validateReadPermission(request) {
              parameters("resource".?, "permission".?) {
                case (resourceName, permissionName) =>
```